.github/workflows/android.yaml

```diff
@@ -24,10 +24,10 @@ jobs:
 
         strategy:
             matrix:
-                type: [arm, arm64]
+                type: [arm, arm64, x64]
 
         env:
-            BUILD_TYPE: ${{ matrix.type }}
+            BUILD_TYPE: android_${{ matrix.type }}
```

Comment:
just curious, where does BUILD_TYPE android_XXX get used?  looks like you removed the only reference to it below on line 48

Reply:
The helper script sets the out dir name from this.